modules/custom/openy_upgrade_tool/openy_upgrade_tool.install

```diff
@@ -204,3 +204,19 @@ function openy_upgrade_tool_update_8008() {
   $config_importer->importConfigSimple('system.action.apply_active_version');
   $config_importer->importConfigSimple('system.action.apply_openy_version');
 }
+
+function openy_upgrade_tool_update_8009() {
+  $config = Drupal::service('config.factory')->getEditable('openy_carnation.settings');
+  $config->set("css_selectors_textarea", ".viewport .page-head__main-menu .nav-level-3 > a
```

Comment:
Lets rename config property to `matchheight_css_classes`
This is more readable.